Dockerfile

```diff
@@ -6,7 +6,7 @@
 
 FROM alpine:latest
 
-MAINTAINER Luc Perkins <lperkins@linuxfoundation.org>
+LABEL maintainer "Luc Perkins <lperkins@linuxfoundation.org>"
 
```

Comment:
@inductor , should the LABEL cmd use an `=` between key/value?
LABEL maintainer=" ... "
/cc @lucperkins

Reply:
@inductor , let's go with LABEL maintainer="..."
https://docs.docker.com/engine/reference/builder/#label

Reply:
@kbhawkey applied!

Reply:
Thanks, @inductor 
/lgtm